configuration/backend_configuration/ampathforms/procedure-history-form.json

```diff
@@ -0,0 +1,174 @@
+{
+  "name": "Procedure Form",
+  "pages": [
+    {
+      "label": "Procedure History",
+      "sections": [
+        {
+          "label": "Procedure History",
+          "isExpanded": "true",
+          "questions": [
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "select",
+                "isSearchable": true,
+                "concept": "1651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "answers": [
+                  {
+                    "concept": "441AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Clean and dressing"
+                  },
+                  {
+                    "concept": "1934AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Suture wound with dressing"
+                  },
+                  {
+                    "concept": "654781a2-5b3d-4fc6-a55b-66e8d7bce2f8",
+                    "label": "Orbital surgery"
+                  },
+                  {
+                    "concept": "646e5c57-c07d-4752-b1b3-dc4e14959419",
+                    "label": "Valvuloplasty"
+                  }
+                ]
+              },
+              "id": "procedurePerformed",
+              "label": "Name of Procedure performed",
+              "required": true
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "datetime",
+                "concept": "160715AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+              },
+              "id": "procedureDate",
+              "label": "Date/time procedure performed",
+              "datePickerFormat": "both"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "datetime",
+                "concept": "167132AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+              },
+              "id": "procedureEndDate",
+              "label": "Procedure end date/time",
```

Comment:
Should we validate this field so that the procedure end date/time cannot be earlier than the procedure date?